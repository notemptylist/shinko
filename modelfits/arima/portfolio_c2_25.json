{
    "numlags": 400,
    "results": [
        {
            "mean": 0.0016912606114312268,
            "order": [
                5,
                1,
                3
            ],
            "rmse": 1.2630221390055019,
            "tstamp": 1640066041.525801
        },
        {
            "mean": 0.0016912606114312268,
            "order": [
                6,
                3,
                0
            ],
            "rmse": 3.1562289368253014,
            "tstamp": 1640066041.525801
        },
        {
            "mean": 0.0016912606114312268,
            "order": [
                5,
                3,
                0
            ],
            "rmse": 3.4524808776780693,
            "tstamp": 1640066041.525801
        }
    ],
    "stream": "portfolio_c2_25.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}