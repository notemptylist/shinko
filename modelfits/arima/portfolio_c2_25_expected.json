{
    "numlags": 400,
    "results": [
        {
            "mean": 0.05110447900073856,
            "order": [
                3,
                0,
                1
            ],
            "rmse": 0.4402386360483492,
            "tstamp": 1640070479.4784462
        },
        {
            "mean": 0.05110447900073856,
            "order": [
                5,
                0,
                3
            ],
            "rmse": 0.4549204371777199,
            "tstamp": 1640070479.4784462
        },
        {
            "mean": 0.05110447900073856,
            "order": [
                5,
                1,
                0
            ],
            "rmse": 0.47972647814830355,
            "tstamp": 1640070479.4784462
        },
        {
            "mean": 0.02514353123349005,
            "order": [
                1,
                1,
                3
            ],
            "rmse": 0.2699877997811625,
            "tstamp": 1640209153.6348825
        },
        {
            "mean": 0.02514353123349005,
            "order": [
                0,
                0,
                0
            ],
            "rmse": 0.28344478221944486,
            "tstamp": 1640209153.6348825
        },
        {
            "mean": 0.02514353123349005,
            "order": [
                2,
                1,
                0
            ],
            "rmse": 0.28636080628040483,
            "tstamp": 1640209153.6348825
        }
    ],
    "stream": "portfolio_c2_25_expected.json",
    "todo": [
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}